src/app/app.component.ts

```diff
@@ -0,0 +1,37 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.css']
+})
+export class AppComponent {
+  title = 'youtube';
+  isFormSubmitted = false;
+  isFiltering = false;
+  filterTerm: undefined | string = undefined;
+  viewsTerm: undefined | boolean = undefined;
+  dateTerm: undefined | boolean = undefined;
```

Comment:
1. I used typing undefined | boolean because display of arrows in UI depends on this values:
undefined - no arrows in UI (near sort on date, views)
true - arrows up
false -arrors down 
2. I added typing